docs/src/pages/demos/cards/SimpleCard.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles, WithStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import { createStyles } from '@material-ui/styles';
```

Comment:
OIs this function still present in `@material-ui/core/styles`? If so please import it from there. If not I have to fix our babel plugin.

Reply:
The function is still present there. I changed the imports.